packages/common/src/stores/AccountController.ts

```diff
@@ -509,14 +508,14 @@ export default class AccountController {
     return this.features;
   }
 
-  private async afterLogin(user: Customer, customerConsents: CustomerConsent[] | null, accessModel: string, shouldSubscriptionReload: boolean = true) {
+  private async afterLogin(user: Customer, customerConsents: CustomerConsent[] | null, shouldReloadSubscription = true) {
     useAccountStore.setState({
       user,
       customerConsents,
     });
 
     await Promise.allSettled([
-      accessModel === ACCESS_MODEL.SVOD && shouldSubscriptionReload ? this.reloadActiveSubscription() : Promise.resolve(),
+      shouldReloadSubscription ? this.reloadActiveSubscription() : Promise.resolve(), // For every accessModal there could be TVOD items, so we always reload the subscription
```

Comment:
In theory, the only call that fixes this for the TVOD offers is the `await this.refreshEntitlements?.();` in the `reloadActiveSubscription` function. 

I don't think we want to call the subscriptions endpoint when no SVOD offer id is defined. But we do want to invalidate the entitlement calls after reloading the subscription.

Reply:
Yes, you're right. Reduced it to just the entitlements call.
I've noticed that AuthVOD + TVOD doesn't work so well yet, but this something I'm investigating from a different ticket.